acct-user/openrct2/openrct2-0.ebuild

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="A user for the OpenRCT2 dedicated server"
+
+ACCT_USER_GROUPS=( "openrct2" )
+ACCT_USER_HOME="/var/lib/openrct2"
```

Comment:
One more (minor) thing, as this is something I learned recently myself. You shouldn't define this, especially when it uses the default. 
https://bugs.gentoo.org/693964#c3
https://github.com/gentoo/devmanual/pull/116#issuecomment-557123600
http://michael.orlitzky.com/articles/gentoo_glep81_user_package_guidelines.xhtml#home-directory-permissions

Reply:
Thank you. That was also new to me. It should not be a problem, since the init script creates anyway `/var/lib/openrct2` with correct permissions. I changed now locally the `acct-user/openrct2` ebuild and the server still works fine after deleting user/group and recreating by ebuild. Will push an update.